ccpp/suites/suite_FV3_RRFS_v1alpha.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="5">
```

Comment:
@llpcarson when you update the submodule pointer for ccpp-physics and revert `.gitmodules`, please change
```
lib="ccppphys" ver="5"
```
to
```
version="1"
```
to match the new syntax in develop. This has no impact on the tests whatsoever, therefore no need to rerun the tests, but it needs to be consistent for the future (`capgen`).